zen-quotes-app/src/app/app.module.ts

```diff
@@ -3,15 +3,45 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { AuthorsComponent } from './authors/authors.component';
```

Comment:
It's a good practice to order imports.

Challenge: can you find a way to automatically order imports via your IDE or via an automated script?

Thanks! 🙏 🙇

Reply:
Thank you very much for the comment and link!🙏
I've added ESLint and Prettier to solve this issue and set up my IDE to use them on save. 
It helped me to clean up my code as well.

Reply:
Yay, awesome, great work setting up ESLint and Prettier! 🏆 🤖 🎉

Reply:
Hooray! 🥳 Thank you very much! I am really grateful for this task! 🙏 
Because I never did these settings before, I had a chance to work only with someone's settings on my previous projects.
Of course, I am not sure if these settings are perfect, but I can improve them while working on the app)

Reply:
Awesome, great work, @IraLysenko! 🏆 💪